test/big-data.js

```diff
@@ -0,0 +1,24 @@
+var test = require('tape')
+var b64 = require('../lib/b64')
+
+test('convert big data to base64', function (t) {
+  var b64str, arr, i, length
+  var big = new Uint8Array(64 * 1024 * 1024)
+  for (i = 0, length = big.length; i < length; ++i) {
+    big[i] = i % 256
+  }
+  b64str = b64.fromByteArray(big)
+  arr = b64.toByteArray(b64str)
+  t.ok(equal(arr, big))
```

Comment:
I tried that it first. It crashed. Why? Because deepEqual looks at keys. So, it generates an array of 89million keys